File: TEMA2/python/mayormenor.py
def mayormenor(a, b, c):
    mayor=a
    menor=a
    #Para sacar el mayor
    if(a > b and a > c):
        mayor=a
    elif(b > a and b > c):
        mayor=b
    else: 
        mayor=c
    #Para sacar el menor
    if(a < b and a < c):
        menor=a
    elif(b < a and b < c):
        menor=b
    else: 
        menor=c
    return f"Menor: {menor} Mayor: {mayor}"

print("********Mayor Menor**********")
print('Introduzca el valor de a: ')
a = int(input())
print('Introduzca el valor de b: ')
b = int(input())
print('Introduzca el valor de c: ')
c = int(input())
print(mayormenor(a,b,c))

File: TEMA2/typescript/mayormenor.ts
import readline from 'node:readline/promises';
import {stdin as input, stdout as output } from 'node:process'
const rl = readline.createInterface({input, output})

function mayormenor(a: number, b: number, c: number){
let menor = a, mayor=a;
//Para sacar el mayor
if(a > b && a > c) mayor=a;
else if(b > a && b > c) mayor=b;
else mayor=c;
//Para sacar el menor
if(a < b && a < c) menor=a;
else if(b < a && b < c) menor=b;
else menor=c;
return `Menor: ${menor} Mayor: ${mayor}`;
}
    
console.log("********Mayor Menor**********")
const a = await rl.question("Introduzca el valor de a: ")
const b = await rl.question("Introduzca el valor de b: ")
const c = await rl.question("Introduzca el valor de c: ")

console.log(mayormenor(a,b,c));
rl.close();